content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,27 @@
+# DVC Experiments Overview
```

Comment:
it can be a good page (still not clear if need a separate one for this though, considering that we have index)

should we do a diagram here with the basic workflow?

should we include cleaning here?

Reply:
> a diagram for the workflow would be nice

Since finalizing that properly may involve including alternative paths (a sort of flow chart) and design work, I vote to make it a follow-up issue (tied to the Exp Versioning release cc @dberenbaum) so we can merge this, if the content is approved in general.